packages/keyring-api/src/api/address.ts

```diff
@@ -0,0 +1,19 @@
+import type { Infer } from '@metamask/superstruct';
+import { object, string } from '@metamask/superstruct';
+
+/**
+ * The resolved address of an account.
```

Comment:
I think it's worth mentioning that the address is resolved from a request and that it will be used by the RPC Router. Otherwise, it can be a bit confusing to understand what this type is used for.

Reply:
https://github.com/MetaMask/accounts/pull/157/commits/9c4ca153fb200535ab017bea55783338ee565016